spring-boot-docs/src/main/asciidoc/howto.adoc

```diff
@@ -1693,7 +1693,10 @@ properties (with the prefix stripped) when the local `EntityManagerFactory` is c
 
 [[howto-configure-hibernate-naming-strategy]]
 === Configure Hibernate Naming Strategy
-Hibernate defines `Physical` and `Implicit` naming strategies. Spring Boot configures
+Since version 5, Hibernate defines `Physical` and `Implicit` naming strategies which you
+can configure in Spring Boot by providing a fully qualified class name to the properties
+`spring.jpa.hibernate.naming.physical-strategy` and
+`spring.jpa.hibernate.naming.implicit-strategy`. Spring Boot configures
```

Comment:
FWIW, I'm in favour of mentioning the properties here. I'd use a list for them. Something like this:

Spring Boot provides properties for configuring Hibernate's naming strategies:

- `spring.jpa.hibernate.naming.physical-strategy`
- `spring.jpa.hibernate.naming.implicit-strategy`

The properties should be configured with the fully qualified class name of a strategy implementation.